learning_journal_basic1/tests.py

```diff
@@ -1,29 +1,162 @@
-import unittest
+import pytest
+import transaction
+import datetime
+from ..views.default import TIME_FORMAT
```

Comment:
I find it a bit hard to believe that if you can do `from .models import x` below, you cannot also do `from .views.default import y` here.